components/ledger/pkg/machine/vm/machine_test.go

```diff
@@ -1557,3 +1616,126 @@ func TestVariablesErrors(t *testing.T) {
 	}
 	test(t, tc)
 }
+
+func TestMachine(t *testing.T) {
+	p, err := compiler.Compile(`
+		vars {
+			account $dest
+		}
+		send [COIN 99] (
+			source = @world
+			destination = $dest
+		)`)
+	require.NoError(t, err)
+
+	t.Run("with debug", func(t *testing.T) {
+		m := NewMachine(*p)
+		m.Debug = true
```

Comment:
use testing.V() maybe?

Reply:
What is it?

Reply:
Here I'm enabling the debug mode of the machine to increase coverage of the `func (m *Machine) tick()` method